packages/vue/src/components/atoms/SfArrow/SfArrow.html

```diff
@@ -1,8 +1,8 @@
 <div class="sf-arrow">
-  <button v-on="$listeners" class="sf-arrow__button" type="button">
+  <button v-on="$listeners" class="sf-arrow__button" type="button" aria-label="Arrow">
```

Comment:
do we have different between arrow back and arrow forward? If so, let put a default value as "arrow" and allow user to customize this?

Reply:
This can be achieved the way it works in the Carousel component:
https://github.com/DivanteLtd/storefront-ui/blob/master/packages/vue/src/components/organisms/SfCarousel/SfCarousel.html#L16-L19

Should I move the `aria-label` up to the parent `div` and have it as a default so it can get overwritten like that?

Reply:
Sorry, totally miss out this message. I meant here is `aria-label` will receive value from user - prop `ariaLabel` maybe? And this prop will have default value as `Arrow`. I prefer to leave it here to make sure a button has a label, rather than put it to the parent :) @jahvi

Reply:
@mayashavin I've added an ariaLabel prop here: https://github.com/DivanteLtd/storefront-ui/pull/544